views/hooks/useParticipantsList.jsx

```diff
@@ -5,25 +5,38 @@ import { SORT_DIRECTION } from '../../constants'
 
 export default function useParticipantsList() {
   const { user, setNotification, setUser } = useOutletContext()
-  const { uid, preferences } = user
+  const { uid, preferences, access } = user
 
   const [initialLoad, setInitialLoad] = useState(true)
+  const [loading, setLoading] = useState(true)
   const [participants, setParticipants] = useState([])
-  const [searchSubjects, setSearchSubjects] = useState([])
-  const [searchOptions, setSearchOptions] = useState([])
+  const [formFilters, setFormFilters] = useState({
+    searchSubjects: [],
+    studies: [],
+    status: undefined,
+  })
+  const [searchOptions, setSearchOptions] = useState({
+    participants: [],
+    studies: access.map((study) => ({ label: study, value: study })),
+  })
   const [sortDirection, setDirection] = useState(SORT_DIRECTION.ASC)
-  const [sortBy, setSortBy] = useState('')
+  const [sortBy, setSortBy] = useState('subject')
```

Comment:
Does it make sense to put `'subject'` in a const?

Reply:
[8574939](https://github.com/AMP-SCZ/dpdash/pull/561/commits/8574939f886757e4a759bd10c0e04266485108cb)